src/docs/faq.md

```diff
@@ -1,70 +1,99 @@
+
+### Can I get AVS or SOF drivers for free?
+* No. Only the Linux drivers are free.
+
+#### Why is there a license error in Device Manager after installing audio drivers?
+* You messed up the license setup. Please refer to [CoolStar's driver portal](https://coolstar.org/chromebook/driverlicense/login.html).
+
+### Will the audio driver still work when re-installing Windows?
 * Yes, on the device the license was created. If the internal drive was replaced, see below.
 
-#### I upgraded my Chromebooks' internal drive and my audio/Thunderbolt 4 license is not working, what do I do?
+### I upgraded my Chromebooks' internal drive and my audio/Thunderbolt 4 license is not working, what do I do?
 * The license is tied to the serial number of your internal drive. **You do not need to buy another license**. Send CoolStar a message through Patreon, and wait for about a week.
 
-#### How do I buy Windows audio drivers in bulk?
+### How do I buy Windows audio drivers in bulk?
 * A discount is only available for Chromebooks containing an `Intel 6th gen processor (up to Core i5)`, `N3350`, `N4000`, or, `N4020` processor, otherwise you must pay full price. You must purchase 10 or more licenses to be eligible for a discount. If you are eligible for a discount, you should ask in the [Chrultrabook Forum](https://forum.chrultrabook.com).
 
 #### Can I use one driver license across multiple devices?
 * No. One license per device.
 
-#### I want to buy a Chromebook to run Windows or Linux or macOS -- which should I buy?
-* Buying any Chromebook with the intention of running macOS, Windows or Linux is not a great idea. -- DO NOT EXPECT EVERYTHING TO WORK OUT OF THE BOX. Older models may fair better compatibility wise, but there are still lots of caveats, and it's not recommended to buy a Chromebook as a cheap Linux device.
+<br>
 
-#### How can I get audio working under Linux?
-* Please see [here](https://github.com/WeirdTreeThing/chromebook-linux-audio)
+## Linux Questions
 
-#### How do I get my top row keys on Linux to act like how they did on chromeOS?
-* Follow the instructions [here](https://github.com/WeirdTreeThing/cros-keyboard-map).
+### How come audio isn't working on Ubuntu / Ubuntu forks?
+* Ubuntu and Ubuntu-based distros may have issues and are not supported, try switching to another distro. Debian is a great alternative if you are used to ubuntu.
 
-#### I just want to boot Linux from USB on my Chromebook, what do I need to do?
-* Check the Supported Devices page, to ensure your device has functional RW_LEGACY firmware available. If so, flash RWL (RW_LEGACY firmware) and use that.
+### I want to buy a Chromebook to run Windows or Linux or macOS -- which should I buy?
+* If you're looking to buy for macOS, you will need a chromebook with an 10th generation intel cpu or lower, Celeron/Pentium are not supported. Any 64 bit chromebook will work in Windows and Linux. Some require paid drivers (Windows only).
```

Comment:
Why specify 10th gen when this is supposed to apply to winders and lienucks too?

Reply:
Its broken into parts.

If you have macOS, you need to have this CPU.

In windows, any 64 bit cpu will work